frontend/src/util/TableCellEditFuncs.tsx

```diff
@@ -107,6 +107,9 @@ export function percentBodyTemplate(value: number) {
 }
 
 export function priceBodyTemplate(value: number | bigint) {
+  if (value == null) {
+    return "-";
```

Comment:
What use case is this actually for? Don't to build in preventative measures that obscure actual issues